requirements-dev.txt

```diff
@@ -31,12 +31,12 @@ lightgbm<4.0.0  # 4.0.0 is incompatible with latest numpy https://github.com/mic
 xlrd
 
 # Required for documentation
-bokeh
 pypandoc
-sphinx==4.5.0
+sphinx==7.1.2; python_version=="3.8"
+sphinx==7.2.6; python_version>"3.8"
```

Comment:
I take it that this is a conditional for a requirements.txt file?

Reply:
Correct. I believe the newer version is not available in 3.8 since sphinx dropped support in (I want to say...) 7.2.